### ISSUE TITLE
Make app titles more descriptive

### ISSUE THREAD

**Issue by lyndseypadget** [tastejs/todomvc#1977]
I'd like to request that the h1 "todos" title of each example include framework-specific info such as "Vue todos" or "React todos".  This simple (but broad) change would be helpful when running/comparing different versions of the Todo app, especially locally.

Potentially related: modify the h1 style to be smaller (to allow for more information) and ensure the HTML title (tab text) matches the h1 title.

**Comment by andrehgdias** [comment]
may I work on this? It's kinda my first open source contribution, but I think I can help with one or two to-dos..

**Comment by lyndseypadget** [comment]
Certainly, as far as I'm concerned!  I'm not a core contributor so I can't speak for the team.

**Comment by FadySamirSadek** [comment]
@andrehgdias @lyndseypadget  Thank you for your interest in contributing to TodoMVC.
 Please open a PR and I'll work with you to get it shipped 🎆

**Comment by andrehgdias** [comment]
Just made a small alteration in React example, I hope it's matching project requirements #2089

**Comment by YukiSakuraVN** [comment]
No action for a long time :(

**Comment by apalevich** [comment]
Is it actual still? I could do that

**Comment by addyosmani** [comment]
Thanks so much for this contribution. I'm going through old issues and closing them out as we last updated the project in a major way in 2024 and want to ensure we're focused on only those issues still relevant. Thanks once again, feel free to reopen if still relevant
